docs/getting-started/create-react-app.mdx

```diff
@@ -1,50 +1,92 @@
 # Create React App
 
-Please note there’s a [known bug][] with
-the macro so live reloading doesn’t
-currently work.
-
-With Create React App you will need to use
-[`mdx.macro`][mdx-macro].
+First, create a new app with [CRA][] and change directory to enter it:
 
 ```sh
 npx create-react-app my-app
-yarn add mdx.macro
+cd my-app
+```
+
+Second, we need to configure some build tools.
+Install [`@mdx-js/loader`][loader] and [`babel-loader`][babel-loader] as dev
+dependencies:
+
+```sh
+yarn add @mdx-js/loader babel-loader --dev
+```
+
+Also make sure that JSX in MDX is handled by Babel by creating a `.babelrc` JSON
+file in the project root of `my-app`:
+
+```json
+{
+  "presets": ["@babel/preset-react"]
+}
+```
+
+Third, we can add our MDX content.
+Create an MDX file `Content.mdx` in the `src/` folder:
+
+```mdx
+export const Box = () => (
+  <div style={{padding: 20, backgroundColor: 'tomato'}} />
+)
+
+# Hello, world!
+
+This is **markdown** with <span style={{color: "red"}}>JSX</span>: MDX!
+
+<Box />
 ```
 
-Then create the following `src/App.js`:
+And to use that MDX content in the app, replace the contents of `App.js` in the
+`src/` folder with:
 
 ```js
-// src/App.js
-
-import React, {lazy, Component, Suspense} from 'react'
-import {importMDX} from 'mdx.macro'
-
-const Content = lazy(() => importMDX('./Content.mdx'))
-
-class App extends Component {
-  render() {
-    return (
-      <div>
-        <Suspense fallback={<div>Loading...</div>}>
-          <Content />
-        </Suspense>
-      </div>
-    )
-  }
+/* eslint-disable import/no-webpack-loader-syntax */
+import Content from '!babel-loader!@mdx-js/loader!./Content.mdx'
+
+function App() {
+  return <Content />
 }
 
 export default App
 ```
 
-And then create the following `src/Content.mdx`:
+We’re almost done!
+To start the development server, run:
 
-```md
-# Hello, world!
+```sh
+yarn start
 ```
 
-[See the full example][cra-example]
+**…but you will probably get this error**:
+
+```txt
+$ react-scripts start
+
+There might be a problem with the project dependency tree.
+It is likely not a bug in Create React App, but something you need to fix locally.
+
+The react-scripts package provided by Create React App requires a dependency:
+
+  "babel-loader": "$VERSION"
+…
+```
+
+CRA expects a certain `babel-loader` version but MDX also needs one.
+To fix the error, make sure the two `babel-loader` versions match, by running
+the following:
+
+```sh
+yarn add babel-loader@$VERSION --dev
+```
+
+(where `$VERSION` is the version number printed in the error message, such as
+`8.1.0`).
+
+Then, running `yarn start` again should do the trick! ✨
```

Comment:
CRA + TS users may still see an error, due to MDX not having a typing.
It may be worth linking to the TypeScript guide for `@mdx-js/loader` as well

Reply:
how about this?